tzrec/models/dbmtl.py

```diff
@@ -31,12 +31,13 @@ class DBMTL(MultiTaskRank):
         model_config (ModelConfig): an instance of ModelConfig.
         features (list): list of features.
         labels (list): list of label names.
+        sample_weights (list): sample weight names.
     """
 
     def __init__(
-        self, model_config: ModelConfig, features: List[BaseFeature], labels: List[str]
+        self, model_config: ModelConfig, features: List[BaseFeature], labels: List[str], sample_weights: List[str] = None
```

Comment:
- add **kwargs to all models for better compatibility
- sample_weights typing is Optional[List[str]]

Reply:
resolved